whois-rpsl/src/main/java/net/ripe/db/whois/common/ip/IpExtractor.java

```diff
@@ -0,0 +1,27 @@
+package net.ripe.db.whois.common.ip;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class IpExtractor {
+
+    private static final Pattern IPV6_SPLIT_PATTERN = Pattern.compile("" +
+            "(" +
+            "[\\dA-Fa-f]{1,4}(?::[\\dA-Fa-f]{1,4}){7}" + // basic case when we get full IPv6(2001:0000:130F:0000:0000:09C0:876A:130B)
+            "|(?:[\\dA-Fa-f]{1,4}:){1,6}(?::[\\dA-Fa-f]{1,4}){1,6}" + // case when there is a double colon in the IPv6(2a00:1f78::fffe)
```

Comment:
What happens if user just send 2001: ? Colon can appear in descr or remarks as well which may or may not be ipv6

Reply:
we need to escape de colon anyways, right? if not we will throw an error